src/04-modality-specific-files/08-genetic-descriptor.md

```diff
@@ -0,0 +1,60 @@
+# Genetic Descriptor
+
+The extension was primarily developped by Cyril Pernet and Clara Moreau with contributions from Tom Nichols and Jessica Turner.
+
+The goal of the genetic descriptor is to link imaging and genetic data by providing minimal information about i) where to find genetic information associated with the imaging data and ii) what type of genetic information is available. This is necessary as genetic data are typically stored in dedicated repositories, separately from the imaging data.
+
+## dataset_description.json
+
+Two additional keys related to the genetic data can be added. The Key `GeneticDataBase` (MANDATORY) links to the name of the database and web address. The key `GeneticDescriptor` (OPTIONAL) refers to the descriptor (e.g. journal article) of the genetic data.
```

Comment:
I suggest to revise this paragraph to use our words MUST(=mandatory), SHOULD(=recommended), and MAY(=optional), see [rfc2119](https://www.ietf.org/rfc/rfc2119.txt).

This would also solve the issue that the first sentence is a bit misleading, because it says "two keys **can** be added", which sounds like both are optional

Reply:
Maybe `GeneticDataBase` could be mandatory if 'GeneticID' is present in any tsv files or if a genetic_info.json file is present.

Reply:
The Key `GeneticDataBase` must be added to link to the name of the database and web address. The key `GeneticDescriptor` may also be present refering to the descriptor (e.g. journal article) of the genetic data.

Reply:
'GeneticID' is not mandatory if the imaging repo and genetic repo use the same ID (which the validator cannot check)